source/options.css

```diff
@@ -1,28 +1,14 @@
-html {
-	min-width: 550px;
-	overflow-x: hidden;
-}
-
-html.is-edgium {
-	width: 100%;
-	min-width: auto;
-}
-
 :root {
 	--github-green: #28a745;
 	--github-red: #cb2431;
 }
 
-body {
-	font-family: system-ui, sans-serif;
-	font-size: 14px;
-	margin: 0;
-	padding: 16px;
+html:not(.is-edgium) {
+	min-width: 550px;
+	overflow-x: hidden;
 }
 
-p {
-	margin-top: 0.25em;
-	margin-bottom: 0;
+body {
 	line-height: 1.5;
```

Comment:
If you want to keep this, you can move it to `:root` so this `body` rule can be dropped.

Reply:
Dropped that anyway.